proof/refine/TcbAcc_R.thy

```diff
@@ -43,17 +39,86 @@ definition
 
 lemma getHighestPrio_def'[simp]:
   "getHighestPrio d = gets (lookupBitmapPriority d)"
-  unfolding getHighestPrio_def gets_def
+  unfolding getHighestPrio_def
   by (fastforce simp: gets_def get_bind_apply lookupBitmapPriority_def bitmap_fun_defs)
 
+locale TcbAcc_R =
+  (* FIXME arch-split: these versions use word_size_bits and can be made generic *)
+  assumes no_fail_loadWord_bits[wp]:
+    "\<And>p. no_fail (\<lambda>_. is_aligned p word_size_bits) (loadWord p)"
+  assumes no_fail_storeWord_bits:
+    "\<And>p w. no_fail (\<lambda>_. is_aligned p word_size_bits) (storeWord p w)"
```

Comment:
Is that a TODO for later or did you mean to include it in this PR?

Reply:
I'm not really sure what to do with this comment. I guess I could remove the FIXME from it, but the long-term idea would be to not have `is_aligned p 3` style lemmas running around. To do that would need updating from Machine downwards though, and that's a lot of noise.
For now I'll keep the comment, drop the FIXME.